README.md

```diff
@@ -247,12 +247,18 @@ Add dependencies & initialize ImageEditor class with given element to make an im
 ```javascript
 var ImageEditor = require('tui-image-editor');
 var blackTheme = require('./js/theme/black-theme.js');
+var locale_ru = { // override default English locale to your custom
```

Comment:
I think we need a document of a full key list. How about [/docs/Basic-Tutorial.md](https://github.com/nhnent/tui.image-editor/blob/master/docs/Basic-Tutorial.md)? 
@jinwoo-kim-nhn Anh idea?

Reply:
@dongsik-yoo, I will make doc today.